### ISSUE TITLE
error : unexpected key in source state_dict

### ISSUE THREAD

**Issue by hotelbread** [ViTAE-Transformer/ViTPose#67]
Thank you for your effort. Im very glad to use your model. 
But Im not able to train the Vit-small model by my own data.
Actually, when I run the code, I met this one
![image](https://user-images.githubusercontent.com/91708227/212862724-ff3dc09e-a8a5-48c6-ab2b-7603005bdde4.png)

![image](https://user-images.githubusercontent.com/91708227/212862981-1ea1dc44-1801-469a-9841-24df31e1a476.png)

How can I fix this error??

**Comment by Annbless** [comment]
It seems that you load the pose-trained model via the ```pretrained``` option. If you want to train the small-size model, please download and use the MAE pre-trained small-size model via the ```pretrained``` option. The MAE pre-trained models can be downloaded from [link](https://github.com/ViTAE-Transformer/ViTPose#mae-pre-trained-model).

**Comment by hotelbread** [comment]
Thx!!!!!!!!!!
I tried with MAE pre-trained models and then I got a new warning message. 
![image](https://user-images.githubusercontent.com/91708227/213335451-4a3cdd34-d28f-4ea2-ae1b-1c54766851eb.png)

It looks like solving the problem but it still showed the unexpected key and missing key error.
Are that  necessary keys for training??

And I have other question! 
when I training, how can I get a Val-loss and each traing-mAP??
I could only see the train-loss and Val-mAP via log and tensorboard.

Have a good day!

**Comment by Annbless** [comment]
The unexpected keys are the decoders in MAE, which are not necessary for the training in ViTPose. I think the val loss is available in the log file. The training mAP is not naturally supported in the code.

**Comment by hotelbread** [comment]
Thank you!!!
So, missing keys are also not necessary for training as well? Then I could train by my own data!!! 
My work would be just finding how to print a val loss.

**Comment by Annbless** [comment]
Yes. The missing keys are not necessary. I will close this issue temporarily. If you have some further questions, please feel free to re-open it.

**Comment by eliabntt** [comment]
Hi @Annbless. What if I want to use a pre trained ViT model? How can I freeze the backbone and train just the new head on my dataset?
